### PR TITLE
Add a blog post launching project director elections

### PR DESCRIPTION
**Pull request by eholk**
Add a post describing the Project Director election process as defined at https://github.com/rust-lang/leadership-council/pull/12. This also serves as the official launch of the process.

I set the publish date at September 1, since this is the latest we can kick off the process and still finish in time, but if we get all the approvals sooner we should publish then.

## Checkboxes

* [x] @Mark-Simulacrum
* [x] @carols10cents
* [x] @eholk
* [x] @ehuss
* [x] @jackh726
* [x] @jonathanpallant
* [x] @technetos 
* [x] @m-ou-se
* [x] @rylev

### CODE REVIEW

**Comment by thejpster** [comment]
Looks good to me. Will check the @jonathanpallant box at work tomorrow.

File: posts/2023-09-01-electing-new-project-directors.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Today we are launching the process to elect new Project Directors to the Rust Foundation Board of Directors.
+As we begin the process, we wanted to spend some time explaining the goals and procedures we will follow.
+We will summarize everything here, but if you would like to you can read the [official process documentation][pde-process].
+
+We ask all project members to begin working with their Leadership Council representative to nominate potential Project Directors. See the [Candidate Gathering](#candidate-gathering) section for more details. Nominations are due by September 15, 2023.
+
+<!-- TODO: Once we've merged this PR we should link to the final location instead. -->
+[pde-process]: https://github.com/rust-lang/leadership-council/pull/12
+
+The Rust Foundation Board of Directors has five seats reserved for Project Directors.
```
Comment (rylev) [inline posts/2023-09-01-electing-new-project-directors.md:17]: Nit: Perhaps a section header of "What is the Project Director Role" would make things flow more nicely?

**Review by rylev — APPROVED** [review]
Looks great!!! Before we merge, we should do one final pass at the dates to make sure they still make sense.

File: posts/2023-09-01-electing-new-project-directors.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Candidate Gathering
+
+The first phase is beginning right now.
+In this phase, we are inviting the members of the Rust Project to nominate people who will make good project directors.
+The goal is to bubble these up to the Council through each of the top-level teams.
+You should be hearing from your Council Representative soon with more details, but if not, feel free to reach out to them directly.
+
+Each team is encouraged to suggest candidates.
+Since we are electing two new directors, it would be ideal for teams to nominate two candidates, but teams can nominate more or fewer as they see fit.
```
Comment (Mark-Simulacrum) [inline posts/2023-09-01-electing-new-project-directors.md:35]: This feels iffy to me -- I think we should say *at least* two candidates. We're somewhat worried about not having enough candidates, I definitely don't want a team to think two is a quota they shouldn't exceed.

Comment (eholk) [reply posts/2023-09-01-electing-new-project-directors.md:35]: Sounds good. I changed it to say at least two, and removed the part that said more or fewer is okay. I figure since it still says "it's ideal" we're not making it a requirement, but we're also not highlighting the fact that teams aren't required to nominate anyone.

File: posts/2023-09-01-electing-new-project-directors.md (hunk 1 above)
Comment (eholk) [reply posts/2023-09-01-electing-new-project-directors.md:17]: Done!

**Comment by eholk** [comment]
@rust-lang/leadership-council - I added an additional paragraph thanking the people involved in creating this process. Let me know if this sounds good or we should make any changes!

File: posts/2023-09-01-electing-new-project-directors.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+We will summarize everything here, but if you would like to you can read the [official process documentation][pde-process].
+
+We ask all project members to begin working with their Leadership Council representative to nominate potential Project Directors. See the [Candidate Gathering](#candidate-gathering) section for more details. Nominations are due by September 15, 2023.
+
+<!-- TODO: Once we've merged this PR we should link to the final location instead. -->
+[pde-process]: https://github.com/rust-lang/leadership-council/pull/12
+
+## What are Project Directors?
+
+The Rust Foundation Board of Directors has five seats reserved for Project Directors.
```
Comment (carols10cents) [inline posts/2023-09-01-electing-new-project-directors.md:19]: Some context that people reading this might appreciate is how many seats total there currently are on the council (iirc it depends on how many corporate members at the top level?)

I'm willing to be overridden if others think this isn't relevant here.

File: posts/2023-09-01-electing-new-project-directors.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+These Project Directors serve as representatives of the Rust project itself on the Board.
+Like all Directors, the Project Directors are elected by the entity they represent, which in the case of the Rust Project means they are elected by the Rust Leadership Council.
+Project Directors serve for a term of two years and will have staggered terms.
+This year we will appoint two new directors and next year we will appoint three new directors.
```
Comment (carols10cents) [inline posts/2023-09-01-electing-new-project-directors.md:23]: Is it ok if we put the names of the directors that are stepping down this year into this post? (and maybe remind people who the other three are too?) I think that'll be one of the questions people reading this immediately have, but if there's a reason we don't want to put that info in here, that's fine.

File: posts/2023-09-01-electing-new-project-directors.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+We want to make sure the Project Directors can effectively represent the project as a whole, so we are soliciting input from the whole project.
+The elections process will go through two phases: Candidate Gathering and Election.
+Read on for more detail about how these work.
+
+## Candidate Gathering
+
+The first phase is beginning right now.
+In this phase, we are inviting the members of the Rust Project to nominate people who will make good project directors.
```
Comment (carols10cents) [inline posts/2023-09-01-electing-new-project-directors.md:32]: Given that one of our goals is to clarify what "members of the Rust Project" means, I think we should say explicitly here that we're meaning "members of a Rust team"

**Review by carols10cents — COMMENTED** [review]
Just some tiny suggestions for possible clarifications; I don't consider any of them blockers, nor do I feel the need to re-review if these suggestions are taken or not, so I'm going to check my box and please consider it durable through minor changes.

File: posts/2023-09-01-electing-new-project-directors.md (hunk 3 above)
Comment (Mark-Simulacrum) [reply posts/2023-09-01-electing-new-project-directors.md:19]: The project directors have a fixed 5 seats. There's no automatic growth with growth of the overall board, but any vote requires a majority of both project & corporate directors, so the growth doesn't dilute vote power for the project.

I'm not sure getting into this level of nuance is hugely helpful here - it's usually a few hoop jumps for folks to understand but gets resolved quickly in comment threads etc. But I'm not opposed.

Comment (eholk) [reply posts/2023-09-01-electing-new-project-directors.md:19]: I think at least saying something like "we're rotating out two of the five project director seats" would be helpful and not too much detail. I'll add something about this.

File: posts/2023-09-01-electing-new-project-directors.md (hunk 4 above)
Comment (eholk) [reply posts/2023-09-01-electing-new-project-directors.md:23]: I can't find it in Zulip, but I raised this at one point and @rylev (I think, I could be misremembering) suggested we should wait to say that when we announce the results of the election. But, all the comments I could find from people say the opposite, so maybe I'll include something.

File: posts/2023-09-01-electing-new-project-directors.md (hunk 3 above)
Comment (eholk) [reply posts/2023-09-01-electing-new-project-directors.md:19]: Oh, I guess I already had the number of project directors already in there..

I think I'd rather not have the extra detail about total number of seats, growth of the board, and the way voting power is allocated. The whole thing seems like more detail than is really appropriate for this post and anything less than that seems like not enough to stand on its own.

File: posts/2023-09-01-electing-new-project-directors.md (hunk 5 above)
Comment (eholk) [reply posts/2023-09-01-electing-new-project-directors.md:32]: I'll reword this as "members of all of the top level Rust teams and their subteams"

**Comment by eholk** [comment]
Alright, it looks like we have all of our approvals (@technetos told me out-of-band that they approve, but if you want you can check your box here now too). I'm going to plan on publishing this tomorrow afternoon in my timezone. I made a few new edits based on review suggestions (the most significant one is listing the names of the current PDs and those who are rotating out), so if there are any objections please let me know.

File: posts/2023-09-01-electing-new-project-directors.md (hunk 4 above)
Comment (eholk) [reply posts/2023-09-01-electing-new-project-directors.md]: I added the names of the current project directors as well as the ones who will be rotating out.
